### DISCUSSION TITLE
Qi deck rejected by belkin charger

### DISCUSSION THREAD

**Discussion by wydmynd** [bitcraze/discussions#158] [Q&A]
Hello
I am testing the Qi deck as standalone charger
I hooked up wires from a Qi deck GND/VUSB to a micro USB cable
and plugged it into a small device (in this case a small Miracase power bank)
when putting it on a belkin F8M747 charger, I am getting errors from the charger (red led blinks) and the charging stops. 
- when the Qi deck is not connected to anything, sometimes it gets rejected and sometimes not
- when a device is consuming power, it always gets rejected
- tried connecting a 22uf cap to GND/VUSB but same results

**Comment by krichardsson** [comment]
@evoggy or @tobbeanton are the best persons to answer this question but unfortunately they are currently enjoying their vacations and will not be back until August 15.

**Reply by wydmynd** [reply]
thanks, I will be happy to hear from them when they get back, meanwhile I will try to obtain a different module

**Reply by tobbeanton** [reply]
Do you have the old Qi-deck or Qi 1.2 deck? The old Qi-deck is not compatible with the 1.2 standard which might be the problem. If so I think it can be patched to work with the 1.2 standard.
